### ISSUE TITLE
Expectation for generated typings file

### ISSUE THREAD

**Issue by spjtls9** [TeamSupercell/typings-for-css-modules-loader#44]
Hi-

First of all, thanks for the fork - it is very helpful to have an active repo for this.

Question:

Am I supposed to check in the generated typings file to source control?

There is this issue in the old repo: https://github.com/Jimdo/typings-for-css-modules-loader/issues/79#issue-388133750
And without checking in the typings files, the same issue will happen. 

Is there any known issues with checking in the typings file?

**Comment by monic-shixi** [comment]
> Hi-
> 
> First of all, thanks for the fork - it is very helpful to have an active repo for this.
> 
> Question:
> 
> Am I supposed to check in the generated typings file to source control?
> 
> There is this issue in the old repo: [Jimdo/typings-for-css-modules-loader#79 (comment)](https://github.com/Jimdo/typings-for-css-modules-loader/issues/79#issue-388133750)
> And without checking in the typings files, the same issue will happen.
> 
> Is there any known issues with checking in the typings file?

Yep, maybe the https://github.com/TeamSupercell/typings-for-css-modules-loader#typescript-does-not-find-the-typings and you are used fork-ts-checker-webpack-plugin?
